packages/contentful/migrations/crn/manuscripts/20240522140622-create-manuscript-version-model.js

```diff
@@ -0,0 +1,83 @@
+module.exports.description =
+  'Add fields type and lifecycle to manuscripts content model';
+
+module.exports.up = (migration) => {
+  const manuscriptVersions = migration
+    .createContentType('manuscriptVersions')
+    .name('Manuscript Versions')
+    .description('')
+    .displayField('type');
+
+  manuscriptVersions
+    .createField('type')
+    .name('Type')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
```

Comment:
maybe we want to have it as required in the CMS as well? 🤔 the same thing for the lifecycle field